src/app/views/onboarding/onboarding.html

```diff
@@ -0,0 +1,43 @@
+<div id="wizard">
+  <div class="main-box">
+    <wizard on-finish="finishedWizard()">
+      <wz-step wz-title="Select your application">
+        <mno-welcome-wizard></mno-welcome-wizard>
+        <div class="text-right">
+          <a href="#" wz-next class="btn btn-default btn-lg">{{'mno_enterprise.templates.onboarding.mno_welcome_wizard.next' | translate}}<span class="glyphicon glyphicon-menu-right"></span></a>
+        </div>
+      </wz-step>
+      <wz-step wz-title="connect your application">
+        <div class="text-center">
+          <h1>1. {{'mno_enterprise.templates.onboarding.select_your_app.select_application' | translate}}</h1>
```

Comment:
I think you should include the `1.` in the locale string (arabic/chinese languages may do otherwise)